src/api/listing/service.proto

```diff
@@ -210,6 +210,9 @@ message BeginIngestionResponse {}
 
 // RequestIngestionRequest is used to request ingestion of a listing into Zentail
 message RequestIngestionRequest {
+  // An identifier for this listing that will then be used for the Product Group Label
+  // e.g. Amazon Parent SKU or Target Parent Variation Product (aka VAP) SKU
```

Comment:
my only concern with this is given this is the public api-proto repo maybe the amazon parent sku / target vap is not contextual here?

Reply:
ok I just removed that line